docs/rc.xml.all

```diff
@@ -394,14 +394,28 @@
   </mouse>
 
   <!--
+    The responsive tablet area can be specified by setting the lop/left
+    and/or width/height coordinates (in mm). This allows, among others,
+    to set the tablet area aspect ratio to the aspect ratio of the
+    output.
+
+    Example: 
```

Comment:
I've reworked the description. it should clarify now how this works instead of how i use it. Does it makes sense now?
The aspect ratio of the tablet is not maintained, that was just my own use case (to change the active area that the aspect ratio from tablet changes from 16:9 to 21:9 so that it matches the one from my monitor)